common/tandemClient.js

```diff
@@ -74,6 +74,26 @@ export class TandemClient {
         return response.headers.get('x-dt-access-level');
     }
 
+    /**
+     * Creates link to upload file.
+     * 
+     * @param {string} facilityId 
+     * @param {string} fileName 
```

Comment:
Parameter `fileName` does not exist. Instead `input` called further down provides something like
```
    {
        name: uploadLink.name,
        fileName: uploadLink.fileName,
        key: uploadLink.key,
        url: uploadLink.url
    }
```